content.js

```diff
@@ -21,18 +21,24 @@ function copyPrDescription() {
   messageField.value = commitBody;
 }
 
-function addMergeListeners() {
-  const squashButton = document.querySelector('.merge-message .btn-group-squash');
-  const mergeButton = document.querySelector('.merge-message .btn-group-merge');
+function addMergeListener(event) {
+  if (!window.location.pathname.match("/pull/[0-9]+$")) return;
 
-  if (squashButton) {
-    squashButton.addEventListener('click', copyPrDescription);
-  }
-  if (mergeButton) {
-    mergeButton.addEventListener('click', copyPrDescription);
-  }
+  const prMergePanel = document.querySelector('.js-merge-pr:not(.is-rebasing)');
+  if (!prMergePanel) return;
 
+  prMergePanel.addEventListener('details:toggled', copyPrDescription);
```

Comment:
Should we try to avoid adding the same handler multiple times? Looks like Refined GitHub uses memoization for that purpose

Reply:
I believe `addEventHandler` handles this.

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Multiple_identical_event_listeners

Reply:
> > Hmmm well I just tried it out and it seems to work, but it's hard to tell since it was also working intermittently before...
> 
> Well I know it at least fixes the new comments issue 
> 
> I can leave the issue up for a bit after merging this for good measure.

Up to you :smile: Can always reopen if it reoccurs.